docs/mockwbemserver.rst

```diff
@@ -199,31 +200,32 @@ The mock WBEM server does NOT support:
 
 Diagram of flow of requests operations through mocker
 
-.. _`Faked WBEM operations`:
 
-Faked WBEM operations
----------------------
+.. _`WBEM operations of a mock WBEM server`:
+
+WBEM operations of a mock WBEM server
+-------------------------------------
 
-The :class:`pywbem_mock.FakedWBEMConnection`  class supports the same WBEM
+The :class:`pywbem_mock.FakedWBEMConnection` class supports the same WBEM
 operations that are supported by the :class:`pywbem.WBEMConnection` class and
-in addition a set of methods to execute WBEM server responders for each of these
-client methods using an internal CIM repository of CIM classes, CIM instances,
-and CIM qualifier declarations.
+in addition a set of methods for setting up its mocked CIM repository, and
+for registering user-defined WBEM operation responders ("providers") for
+some the WBEM operations.
```

Comment:
Change "some the" to something like selected

Reply:
DONE